### PR TITLE
Document `auto_repack` for the recorder

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Document `auto_repack` for the recorder

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/69314
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/recorder.markdown [hunk 1]
```diff
@@ -63,6 +63,11 @@ recorder:
       required: false
       default: true
       type: boolean
+    auto_repack:
+      description: Automatically repack the database every second sunday after the auto purge. Without a repack, the database may not decrease in size even after purging, which takes up disk space and can make Home Assistant slow. If you disable `auto_repack` it is recommended that you create an automation to call the [`recorder.purge`](#service-purge) periodically. This flag has no effect if `auto_purge` is disabled.
```
Comment (bdraco) [inline source/_integrations/recorder.markdown:67]:
```suggestion
      description: Automatically repack the database every second sunday after the auto purge. Without a repack, the database may not decrease in size even after purging, which takes up disk space and can make Home Assistant slow. If you disable `auto_repack` it is recommended that you create an automation to call the [`recorder.purge`](#service-purge) periodically. This flag has no effect if `auto_purge` is disabled or the database engine is not sqlite.
```

Comment (bdraco) [reply source/_integrations/recorder.markdown:67]: https://github.com/home-assistant/core/pull/69331

Comment (bdraco) [reply source/_integrations/recorder.markdown:67]: Resolving this. Will open a new PR if we decide we need to move forward with https://github.com/home-assistant/core/pull/69331

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍

**Pull request merged**